tests/test_exceptions.py

```diff
@@ -244,3 +244,35 @@ def event(self, ev):
         qapp.processEvents()
 
     assert [str(e) for (t, e, tb) in exceptions] == ['mistakes were made']
+
+
+def test_exceptions_to_stderr(qapp):
```

Comment:
Please use the [capsys](http://pytest.org/latest/capture.html?highlight=capsys#accessing-captured-output-from-a-test-function) fixture instead, so you will avoid having to manually override `sys.stderr`.